lib/src/mockserver/server.spec.ts

```diff
@@ -147,5 +148,25 @@ describe("Server", () => {
           done();
         });
     });
+
+    it("Strip draft in request paths", done => {
+      const { app } = runMockServer(contract, {
+        logger: mockLogger,
+        pathPrefix: "/api",
+        port: 8085,
+        proxyConfig: {
+          protocol,
+          proxyBaseUrl
+        }
+      });
+
+      request(app)
+        .post("/api/_draft/companies")
+        .expect(201)
+        .then(response => {
+          expect(response.body.name).not.toBe("This is the real response");
```

Comment:
maybe also check that it is indeed a string?

Reply:
good point, it's returning undefined, will need to figure out why it's not returning anything.